pages/insight_vacancies.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+// Gives the page a title
+setTitle("Vacancies");
```

Comment:
Dit is wat de gebruiker te zien krijgt, dus moet het waarschijnlijk in het Nederlands.